{{cookiecutter.repo_name}}/fabfile.py

```diff
@@ -0,0 +1,249 @@
+"""
+This is a collection of useful utility functions when working with docker on different environments.
+
+In order to use these functions, install fabric on your local machine with::
+
+    pip install fabric
+
+Please note: Fabric is a remote code execution tool, NOT a remote configuration tool. While you can copy files
+from here to there, it is not a good replacement for salt or ansible in this regard.
+
+There is a function called `production` where you need to fill in the details about your production machine(s).
+
+You can then run::
+
+    fab production status
+
+to get the status of your stack
+
+To list all available commands, run::
+
+    fab -l
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+from fabric.operations import local as lrun, run, sudo
+from fabric.api import *
+from fabric.colors import green, red, yellow
+from fabric.contrib.console import confirm
+
+
+def local():
+    """
+    Work on the local environment
+    """
+    env.compose_file = "dev.yml"
+    env.project_dir = "."
+    env.run = lrun
+    env.cd = lcd
+
+
+def production():
+    """
+    Work on the production environment
+    """
+    env.hosts = [""]  # list the ip addresses or domain names of your production boxes here
+    env.port = 22  # ssh port
+    env.user = "root"  # remote user, see `env.run` if you don't log in as root
+
+    env.compose_file = "docker-compose.yml"
+    env.project_dir = ""  # this is the project dir where your code lives on this machine
+
+    # if you don't use key authentication, add your password here
+    # env.password = "foobar"
+    # if your machine has no bash installed, fall back to sh
+    # env.shell = "/bin/sh -c"
+
+    env.run = run  # if you don't log in as root, replace with 'env.run = sudo'
+    env.cd = cd
+
+
+def rollback(commit="HEAD~1"):
+    """
+    Rollback to a previous commit and build the stack
+    :param commit: Commit you want to roll back to. Default is the previous commit
+    """
+    with env.cd(env.project_dir):
+        env.run("git checkout {}".format(commit))
+
+    docker_compose("build")
+
+
+def deploy():
+    """
+    Pulls the latest changes from master, rebuilt and restarts the stack
+    """
+    with env.cd(env.project_dir):
+        env.run("git pull origin master")
+
+    build()
+    restart()
+
+
+def scale(service, n):
+    """
+    Scale a service
+    :param service: Service to scale
+    :param n: Number of containers
+    """
+    if service not in ["celeryworker"]:
+        print(red("{} is not scalabale, aborting".format(service)))
+        return
+    docker_compose("scale {service} {n}".format(service=service, n=n))
+
+
+def restart():
+    """
+    Restart all services
+    """
+    docker_compose("restart")
+
+
+def build(cache=True):
+    """
+    Builds the the stack
+    :param cache: Use docker cache. Default is True
+    """
+    docker_compose("build" if cache else "build --no-cache")
+
+
+def status():
```

Comment:
I think `ps` would be easier to type/remember. But this is minor; I'm fine with leaving it as-is if you prefer.